epoch/api/account_api_usage.md

```diff
@@ -21,7 +21,7 @@ You shall read output like the following:
 
 In order to retrieve your balance, fetch your public key then use it to get the balance associated to that public key (replace the public key in the command):
 ```bash
-curl -G 'http://127.0.0.1:3113/v2/account/balance/ak$3N1WLMewMQPUyQBdEhXRSYee84RQNKJrECwbbseMkNsZhv1XLjpmiqjAkvSRpQ6kgWJMjq9dTmdQ3ekuhpscJk6LpjJYk4'
+curl -G 'http://127.0.0.1:3013/v2/account/balance/ak$3N1WLMewMQPUyQBdEhXRSYee84RQNKJrECwbbseMkNsZhv1XLjpmiqjAkvSRpQ6kgWJMjq9dTmdQ3ekuhpscJk6LpjJYk4'
```

Comment:
Please add port assumption at top of document e.g. https://github.com/aeternity/protocol/blob/28d6eae0ec273a7a6c9a1c07272ff31ad12765b4/epoch/api/naming_system_api_usage.md#introduction

Reply:
Done